lesson04/assignment/css/_components.content.scss

```diff
@@ -0,0 +1,72 @@
+
+// container to hold all the squares
+.container {
+  display: no-flex;
```

Comment:
We don't have a `no-flex` option for `display` -- just `block`, `inline`, `inline-block`, `grid`, and `flex`.